vars/runGradleCheck.groovy

```diff
@@ -51,7 +51,7 @@ void call(Map args = [:]) {
                 ./gradlew --stop
                 rm -rf ~/.gradle
 
-                if ! (uname -s | grep -i NT); then
+                if command -v docker > /dev/null; then
                     echo "Check existing dockercontainer"
                     docker ps -a
                     docker stop `docker ps -qa` > /dev/null 2>&1 || echo
```

Comment:
should be use `force` to make sure all the stale containers are removed?
@peterzhuamazon

Reply:
I will add force but force is more or less trying to rm containers that is still not stopped, considering I already try once to stop the container, this should not be needed but anyway nice to have.

Thanks.